### PR TITLE
chore: removing wrong translations about {{consoleUrl}}

### LINKED ISSUE #13771: [CAL-3184] {{consoleUrl}} and {{supportMail}} not rendering correctly (in Portugese, maybe more languages)
Hello everyone, I installed [cal.com](http://cal.com) today on my server and I need to send requests through an integration to the tool, but when trying to add the API it generates this error, does anyone know how to solve it?

To enable this feature, get a development key from the {{consoleUrl}} console and add it to your .env as CALCOM_LICENSE_KEY. If your team already has a license, contact {{supportMail}} for help.

image

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3184](https://linear.app/calcom/issue/CAL-3184/consoleurl-and-supportmail-not-rendering-correctly-in-portugese-maybe)</sub>

### PR DESCRIPTION
**Pull request by PeerRich**
removing wrong translations will trigger Crowdin to re-translate the correct source string

### CODE REVIEW

File: apps/web/public/static/locales/en/common.json [hunk 1]
```diff
@@ -981,7 +981,7 @@
   "verify_wallet": "Verify Wallet",
   "create_events_on": "Create events on",
   "enterprise_license": "This is an enterprise feature",
-  "enterprise_license_description": "To enable this feature, have an administrator go to <2>/auth/setup</2> to enter a license key. If a license key is already in place, please contact <5>{{SUPPORT_MAIL_ADDRESS}}</5> for help.",
+  "enterprise_license_description": "To obtain a license for this feature, visit https://cal.com/sales. If a license key is already in place, please contact support@cal.com for help.",
```
Comment (Udit-takkar) [inline apps/web/public/static/locales/en/common.json:984]: This won't work with 

Screenshot 2024-02-21 at 7 23 03 PM

Comment (Udit-takkar) [reply apps/web/public/static/locales/en/common.json:984]: Plus https://cal.com/sales would display as text not clickable link

Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:984]: ah yikes. can you fix this?

Comment (PeerRich) [reply apps/web/public/static/locales/en/common.json:984]: great catch

**Comment by Udit-takkar** [comment]
Other translations should also be of the same format as english

"enterprise_license_description": "To enable this feature, have an administrator go to <2>/auth/setup</2> to enter a license key. If a license key is already in place, please contact <5>{{SUPPORT_MAIL_ADDRESS}}</5> for help.",


Screenshot 2024-02-22 at 2 36 02 AM

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**